### PR TITLE
Wayfarer revolver

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
New exl revolver, caseless 10x24 that holds 8 shots. Well not marked its still an exl gun hand made and all that.

### CODE REVIEW

**Review by Yawet330 — CHANGES_REQUESTED** [review]
Extreme spelling mistakes.

"exstreamly"
"Well" should be while

File: code/modules/projectiles/guns/projectile/revolver/wayfarer.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//Sprite by Rebel
+/obj/item/gun/projectile/revolver/wayfarer
+	name = "\"Wayfarer\" caseless revolver"
+	desc = "A exstreamly well hand made revolver of a mix of steel, iron and plasteel with a polished wooden stock. \
+	The barrel inside it refined to be ideal for 10x24 caseless and easy to clean. \
+	Well produced by Excelsior it dosnt have the normal markings other then the quote - \
```
Comment (Yawet330) [inline code/modules/projectiles/guns/projectile/revolver/wayfarer.dm:6]: Here

File: code/modules/projectiles/guns/projectile/revolver/wayfarer.dm [hunk 2]
```diff
@@ -0,0 +1,24 @@
+//Sprite by Rebel
+/obj/item/gun/projectile/revolver/wayfarer
+	name = "\"Wayfarer\" caseless revolver"
+	desc = "A exstreamly well hand made revolver of a mix of steel, iron and plasteel with a polished wooden stock. \
```
Comment (Yawet330) [inline code/modules/projectiles/guns/projectile/revolver/wayfarer.dm:4]: Here

File: code/modules/projectiles/guns/projectile/revolver/wayfarer.dm [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	The barrel inside it refined to be ideal for 10x24 caseless and easy to clean. \
```
Comment (Yawet330) [inline code/modules/projectiles/guns/projectile/revolver/wayfarer.dm:5]: "The Barrel is refined to use high-power 10x24 caseless munitions while being easy to clean."

File: code/modules/projectiles/guns/projectile/revolver/wayfarer.dm [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	Well produced by Excelsior it dosnt have the normal markings other then the quote - \
+	\"The hopeless don't revolt, because revolution is an act of hope.\""
```
Comment (Yawet330) [inline code/modules/projectiles/guns/projectile/revolver/wayfarer.dm:7]: "Revolution is an act of hope, thus it falls upon those willing to take up arms, to save those who will not."

File: code/modules/projectiles/guns/projectile/revolver/wayfarer.dm (hunk 1 above)
Comment (Yawet330) [reply code/modules/projectiles/guns/projectile/revolver/wayfarer.dm:6]: "While seemingly produced by Excelsior; hinted by the stamp placed upon its handguard, it lacks most of the normal markings - and is instead tagged with a simple quote -"

File: code/modules/projectiles/guns/projectile/revolver/wayfarer.dm (hunk 2 above)
Comment (Yawet330) [reply code/modules/projectiles/guns/projectile/revolver/wayfarer.dm:4]: "A sleek; well-made revolver, hand-produced with reinforcements of plasteel, while the barrel assembly was a mix of high-carbon steel; low-carbon iron around the cylinder - and a polished mahogany handle."

File: code/modules/projectiles/guns/projectile/revolver/wayfarer.dm (hunk 4 above)
Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/revolver/wayfarer.dm:7]: This is a real quote

Comment (Yawet330) [reply code/modules/projectiles/guns/projectile/revolver/wayfarer.dm:7]: was a suggestion, closing anyway

**Review by Yawet330 — APPROVED** [review]
russian grammar fixed, and the revolver itself is fine and balancedTM

**Pull request merged**